drv/gimlet-seq-server/src/main.rs

```diff
@@ -832,15 +832,22 @@ impl<S: SpiServer> ServerImpl<S> {
                 //
                 // And establish our timer to check SP3_TO_SP_NIC_PWREN_L.
                 //
-                self.deadline = sys_get_timer().now + TIMER_INTERVAL;
-                sys_set_timer(Some(self.deadline), notifications::TIMER_MASK);
+                self.deadline = userlib::set_timer_relative(
+                    TIMER_INTERVAL,
+                    notifications::TIMER_MASK,
+                );
```

Comment:
tiny nit: isn't there a `use userlib::*` in this module?
```suggestion
                self.deadline = set_timer_relative(
                    TIMER_INTERVAL,
                    notifications::TIMER_MASK,
                );
```